modules/cluster-logging-collector-tolerations.adoc

```diff
@@ -1,39 +1,43 @@
 // Module included in the following assemblies:
 //
-// * logging/cluster-logging-collector.adoc
+// * logging/log_collection_forwarding/cluster-logging-collector.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="cluster-logging-collector-tolerations_{context}"]
-= Using tolerations to control the log collector pod placement
+= Using taints and tolerations to control log collector pod placement
 
-You can ensure which nodes the logging collector pods run on and prevent
-other workloads from using those nodes by using tolerations on the pods.
+You can specify which nodes the logging collector pods run on, and prevent other workloads from using those nodes, by configuring taints and tolerations.
 
-You apply tolerations to logging collector pods through the `ClusterLogging` custom resource (CR)
-and apply taints to a node through the node specification. You can use taints and tolerations
-to ensure the pod does not get evicted for things like memory and CPU issues.
+You apply tolerations to logging collector pods by modifying the `ClusterLogging` custom resource (CR), and apply taints to a node through the node specification. You can use taints and tolerations to ensure that the pod is not evicted due to memory or CPU issues.
 
-By default, the logging collector pods have the following toleration:
+By default, logging collector pods have the following toleration:
 
 [source,yaml]
 ----
-tolerations:
-- key: "node-role.kubernetes.io/master"
-  operator: "Exists"
-  effect: "NoExecute"
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+# ...
+spec:
+  collection:
+# ...
+    tolerations:
+    - key: "node-role.kubernetes.io/master"
+      operator: "Exists"
+      effect: "NoExecute"
+# ...
 ----
 
 .Prerequisites
 
-* The Red Hat OpenShift Logging and Elasticsearch Operators must be installed.
+* You have installed the {clo}.
 
 .Procedure
 
 . Use the following command to add a taint to a node where you want logging collector pods to schedule logging collector pods:
 +
```

Comment:
A real scenarios is like below.
1) collector pod can not be schedule to nodes A. For example:  node which has the taint 
"node-role.kubernetes.io/infra"
2) Add tolerations to deploy collector on the nodes  A.

Reply:
We can update the .Procedure as this https://github.com/openshift/openshift-docs/pull/63569#pullrequestreview-1791783300